internal/services/dataprotection/data_protection_backup_vault_resource.go

```diff
@@ -84,6 +74,32 @@ func resourceDataProtectionBackupVault() *pluginsdk.Resource {
 			"tags": tags.Schema(),
 		},
 	}
+
+	// Confirm with the service team that `SnapshotStore` has been replaced with `OperationalStore`.
+	if !features.FourPointOhBeta() {
+		resource.Schema["datastore_type"] = &pluginsdk.Schema{
+			Type:     pluginsdk.TypeString,
+			Required: true,
+			ForceNew: true,
+			ValidateFunc: validation.StringInSlice([]string{
+				string(backupvaults.StorageSettingStoreTypesArchiveStore),
+				"SnapshotStore",
+				string(backupvaults.StorageSettingStoreTypesVaultStore),
+			}, false),
```

Comment:
Is there a reason not to include the new `backupvaults.StorageSettingStoreTypesOperationalStore` here also? Or is there a reason to wait for 4.0 to support it?

Reply:
It's a good idea to include `backupvaults.StorageSettingStoreTypesOperationalStore`, the code has been updated. Could you please take another look?